src/templates/playgroundTemplate.ts

```diff
@@ -16,10 +16,10 @@ db.sales.insertMany([
   { "_id" : 8, "item" : "abc", "price" : 10, "quantity" : 5, "date" : new Date("2016-02-06T20:20:13Z") },
 ]);
 
-// Run a find command.
+// Run a find command to view items sold on April 4th, 2014.
 db.sales.find({ date: { $gte: new Date("2014-04-04"), $lt: new Date("2014-04-05") } });
 
-// Run an aggregation.
+// to view total sales for each product in 2014.
```

Comment:
@mmarcon the "Run an aggregation ..." got dropped here?

Reply:
Oh, not intentional. Nice catch!